README.md

```diff
@@ -8,8 +24,36 @@ $ npm install
 $ npm start
 ```
 
-Currently the site is compiled to static HTML, CSS, and JS where it can run on github pages.
+## Develop Workflow
+
+After successfully building and running the server, go to localhost:8080/
+
+Now start changing files in /src and refresh localhost:8080/ You don't need to rerun the server, although it doesn't hurt to either.
+
+Some server side files need the server to be rerun with npm start.
+
+## Sitemap
+
+```
+/ -> /src/pages/home.js
+/encrypt/ -> /src/pages/encrypt/index.js
+/open-web-fellows/ -> /src/pages/open-web-fellows/overview.js
+/open-web-fellows/info/ -> /src/pages/open-web-fellows/info.js
+/open-web-fellows/fellows/ -> /src/pages/open-web-fellows/fellows.js
+```
+
+## Source Structure
+
+We use npm scripts as the entry point to build our code and run our server. All out npm scripts and npm dependencies are stored in package.json.
+
+Our client side source code lives in /src. Our HTML source is written in react and lives in /src/components/ and /src/pages/. Pages are the main index files, that are fairly specifc, and components are reusable UI elements, that are a bit more generic.
```

Comment:
also note that there is no "HTML source" when React is involved, so being technically correct is important. "Pages are build using React, with the source code for the client-side pages living in `/src/components/ and ..."